mapping_parenting_tech/analysis/prototyping/play_store/fn_tester.py

```diff
@@ -0,0 +1,140 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.6
+#   kernelspec:
+#     display_name: Python 3.8.0 ('mapping_parenting_tech')
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Testing notebook
+# Testing refactoring of functions to manage ids, details and reviews of Play Store apps
```

Comment:
Could add a sentence about how you would expect someone to use this notebook? Or do you actually intend anyone to use it at all (perhaps it was only for when you were refactoring the functions?)

Also: Is it OK that it seems like these test routines are using the same output files that you're using for the final results? (ie, updating them). I guess the most important thing is that we don't overwrite something by accident.